pixeltable/catalog/catalog.py

```diff
@@ -1076,7 +1076,7 @@ def _clear_tv_cache(self, tbl_id: UUID, effective_version: Optional[int]) -> Non
             tv.is_validated = False
             del self._tbl_versions[tbl_id, effective_version]
 
-    def create_replica(self, path: Path, md: list[schema.FullTableMd]) -> None:
+    def create_replica(self, path: Path, md: list[TableVersionMd]) -> None:
```

Comment:
I really don't like the name "TableVersionMd" for this. We already have a schema.TableVersionMd so this creates horrid namespace confusion. Now we have schema.TableVersionMd that's an element of table_version.TableVersionMd!

I'm fine consolidating these classes but let's please call it something else. cc @mkornacker